tests/integration/helpers/helpers.go

```diff
@@ -0,0 +1,38 @@
+package helpers
+
+import (
+	"bytes"
+	"github.com/newrelic/infra-integrations-sdk/log"
+	"os/exec"
+	"strings"
+)
+
+// ExecInContainer executes the given command inside the specified container. It returns three values:
+// 1st - Standard Output
+// 2nd - Standard Error
+// 3rd - Runtime error, if any
+func ExecInContainer(container string, command []string, envVars ...string) (string, string, error) {
```

Comment:
It would be nice to have a test for this function

Reply:
This is a helper function of the integration test, I won't test a test